resources/scss/ceres/views/Templates/AddressSelect/_address-select.scss

```diff
@@ -51,6 +51,11 @@
 	.add-item{
 		position: absolute;
 		
+		.btn {
```

Comment:
better do min-width in the Ceres/resources/scss/ceres/common/_buttons.scss

Reply:
done